tool/tsh/common/tsh.go

```diff
@@ -1261,8 +1261,6 @@ func Run(ctx context.Context, args []string, opts ...CliOption) error {
 	vnetCommand := newVnetCommand(app)
 	vnetAdminSetupCommand := newVnetAdminSetupCommand(app)
 	vnetDaemonCommand := newVnetDaemonCommand(app)
-	vnetInstallServiceCommand := newVnetInstallServiceCommand(app)
```

Comment:
We don't need the install/uninstall commands anymore?

Reply:
yeah change of plans, my plan now is to handle those with a script in the Connect installer, it's really just like 2 shell commands

Reply:
Nice, seems way simpler.